gtk4/src/subclass/editable.rs

```diff
@@ -40,6 +47,58 @@ pub trait EditableImpl: WidgetImpl {
 }
 
 pub trait EditableImplExt: ObjectSubclass {
+    fn is_delegate_property(&self, id: usize) -> bool {
```

Comment:
Instead of such things, you can also always just all `set_property()` / `get_property()`. If they return `true` then it was one of the editable properties, otherwise it's a property to be handled still.

The implementations of the functions already do exactly the same check you're doing here.